src/ec/packages/ec-component-breadcrumb/breadcrumb.html.twig

```diff
@@ -43,7 +43,10 @@
 } %}
 
 {% set _css_class = 'ecl-breadcrumb' %}
-{% set _extra_attributes = ' aria-label="' ~ _navigation_text ~ '" data-ecl-breadcrumb="true"' %}
+{% set _extra_attributes = [
```

Comment:
We were thinking about improving a bit this, and i like this to be an array of objects to be then merged with extra_attributes coming from the "specs" for then being processed all at once. But at the same time i see that this would introduce some more logic into the template, so maybe you could just write one attribute per line as an array item each, here the odd thing is that we have two attrs defined in the first item of the array.. ;)

Reply:
Updated, it's more clean thank you for the review!